packages/manager/modules/pci/src/projects/project/storages/containers/containers.service.js

```diff
@@ -214,27 +215,31 @@ export default class PciStoragesContainersService {
     );
   }
 
-  addContainer(projectId, { archive, containerType, name, region }) {
-    return this.OvhApiCloudProjectStorage.v6()
-      .save(
-        {
-          projectId,
-        },
-        {
-          archive,
-          containerName: name,
-          region: region.name,
-        },
-      )
-      .$promise.then((container) => {
-        let returnPromise = this.$q.resolve();
-        if (containerType === OBJECT_CONTAINER_TYPE_STATIC) {
-          returnPromise = this.setContainerAsStatic(projectId, container);
-        } else if (containerType === OBJECT_CONTAINER_TYPE_PUBLIC) {
-          returnPromise = this.setContainerAsPublic(projectId, container);
-        }
-        return returnPromise;
-      });
+  addContainer(projectId, { archive, containerType, offer, name, region }) {
+    return (offer === OBJECT_CONTAINER_OFFER_HIGH_PERFORMANCE
+      ? this.$http
+          .post(
+            `/cloud/project/${projectId}/region/${region.actualName}/storage`,
```

Comment:
`region.actualName` should be changed to `region.name` once the API is fixed and it does not give the name as PREPROD.SBG

Reply:
Removed :)